provisioner.go

```diff
@@ -323,7 +322,8 @@ func (p *LocalPathProvisioner) createHelperPod(action ActionType, cmdsForPath []
 	hostPathType := v1.HostPathDirectoryOrCreate
 	helperPod := &v1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: string(action) + "-" + name,
+			Name:      string(action) + "-" + name,
+			Namespace: p.namespace,
```

Comment:
This line should be a separate commit since normally we create the helper pod in the default namespace. Btw, why do we need this?

Reply:
Since the helper pod needs to use the newly created configmap (script) which is in the same namespace as provisioner, then it is necessary to create/delete the helper pod with the provisioner's namespace.